src/Datadog.Trace/Agent/Transports/ApiWebRequest.cs

```diff
@@ -2,7 +2,7 @@
 using System.Threading.Tasks;
 using Datadog.Trace.Agent.MessagePack;
 
-namespace Datadog.Trace.Agent
+namespace Datadog.Trace.Agent.Transports
 {
     internal class ApiWebRequest : IApiRequest
```

Comment:
Since you're refactoring the new types to `HttpClientRequest` and `HttpStreamRequest`, could you rename these `ApiWebRequest*` classes to `WebRequest*`?

Reply:
Why for? Would that not cause confusion with the official `WebRequest` class?

Reply:
@zacharycmontoya if we want to rename this can we do so in a separate commit?

Reply:
Sorry for not responding. Yes let's potentially do this in a separate commit. I don't want to hold up the PR for this